source/features/visit-tag.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {ArrowUpRightIcon, CodeIcon} from '@primer/octicons-react';
+import * as pageDetect from 'github-url-detection';
+
+import {branchSelector} from '../github-helpers/selectors.js';
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+import {wrapAll} from '../helpers/dom-utils.js';
+import {buildRepoURL} from '../github-helpers/index.js';
+import {isHasSelectorSupported} from '../helpers/select-has.js';
+
+async function addLink(branchSelector: HTMLButtonElement): Promise<void> {
+	const tag = branchSelector.getAttribute('aria-label')?.replace(/ tag$/, '');
+	if (!tag) {
+		features.log.error(import.meta.url, 'Tag not found in DOM. The feature needs to be updated');
+		return;
+	}
+
+	wrapAll(
+		[
+			branchSelector,
+			<a
+				className="btn px-2 tooltipped tooltipped-se"
+				href={buildRepoURL('releases/tag', tag)}
+				aria-label="Visit tag"
+			>
+				<ArrowUpRightIcon className="v-align-middle"/>
```

Comment:
Not sure if it's the best icon. Context: https://github.com/refined-github/refined-github/issues/6499#issuecomment-1505104968

More icons: https://primer.style/foundations/icons/

Suggestions welcome also after merge